lib/src/Base/Stat/openturns/Isotropic.hxx

```diff
@@ -0,0 +1,100 @@
+//                                               -*- C++ -*-
+/**
+ *
+ *  Copyright 2005-2020 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_ISOTROPIC_HXX
+#define OPENTURNS_ISOTROPIC_HXX
+
+#include "openturns/StationaryCovarianceModel.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+/**
+ * @class Isotropic
+ */
+
+class OT_API Isotropic
+  : public StationaryCovarianceModel
```

Comment:
Please make inheritance from `CovarianceModelImplementation` as `StationaryCovarianceModel` will be dropped